app/users/index/controller.js

```diff
@@ -1,106 +1,101 @@
 import Controller from "@ember/controller";
-import { computed, action } from "@ember/object";
-import { reads } from "@ember/object/computed";
+import { action, set } from "@ember/object";
 import { inject as service } from "@ember/service";
-import { task, timeout, hash } from "ember-concurrency";
-import QueryParams from "ember-parachute";
+import { tracked } from "@glimmer/tracking";
+import { restartableTask, timeout, hash } from "ember-concurrency";
+import { task as trackedTask } from "ember-resources/util/ember-concurrency";
 import moment from "moment";
 
-const UsersQueryParams = new QueryParams({
-  search: {
-    defaultValue: "",
-    replace: true,
-    refresh: true,
-  },
-  supervisor: {
-    defaultValue: null,
-    replace: true,
-    refresh: true,
-  },
-  active: {
-    defaultValue: "1",
-    replace: true,
-    refresh: true,
-  },
-  ordering: {
-    defaultValue: "username",
-    replace: true,
-    refresh: true,
-  },
-});
```

Comment:
The previous `ember-parachute` implementation had the `refresh` and `replace` parameters set to true. Did you have a look what they do? 

I guess we have to add the [`refreshModel` parameter](https://guides.emberjs.com/release/routing/query-params/#toc_opting-into-a-full-transition) on the refactored route.